assets/js/admin/wc-enhanced-select.js

```diff
@@ -131,6 +131,24 @@ jQuery( function( $ ) {
 								} );
 							}
 						});
+					// Keep multiselects ordered alphabetically if they are not sortable.
+					} else if ( $( this ).prop( 'multiple' ) ) {
+						$( this ).on( 'change', function(){
+							var $children = $( this ).children();
+							$children.sort(function(a, b){
+ 								var atext = a.text.toLowerCase();
+       							var btext = b.text.toLowerCase();
+
+       							if (atext > btext) {
+         							return 1;
+       							}
+       							if (atext < btext) {
+         							return -1;
```

Comment:
This is some sort of rendering error in GitHub. [Indentation is correct](https://github.com/woocommerce/woocommerce/blob/d39f4bbe17778818961322e9f96ca22bf005bc0b/assets/js/admin/wc-enhanced-select.js#L142-L148)